src/liquid.rs

```diff
@@ -0,0 +1,43 @@
+use zed::settings::LspSettings;
+use zed_extension_api::{self as zed, serde_json, Result};
+
+struct LiquidExtension;
+
+impl zed::Extension for LiquidExtension {
+    fn new() -> Self {
+        Self
+    }
+
+    fn language_server_command(
+        &mut self,
+        _language_server_id: &zed::LanguageServerId,
+        worktree: &zed::Worktree,
+    ) -> Result<zed::Command> {
+        let path = worktree
```

Comment:
Following the no-config nature of most other extensions, I think the LSP support should take care of the installation of the Shopify CLI for the dev rather than showing an error.